File: src/services/Services.Python/assistant_service.py
class AssistantService:
    def __init__(self, ollama_client):
        if ollama_client is None:
            raise ValueError("Ollama client is required")

        self.ollama_client = ollama_client

    def chat(self, message: str) -> str:
        return self.ollama_client.chat(None, message)

File: src/services/Services.NestJS/assistant.service.ts
import { Injectable } from '@nestjs/common';
import { AssistantServiceInterface } from './assistant.service.interface';
import { OllamaClient } from './ollama.client';

@Injectable()
export class AssistantService implements AssistantServiceInterface {
  constructor(private readonly ollamaClient: OllamaClient) {}

  async chat(message: string): Promise<string> {
    return await this.ollamaClient.chat(undefined, message);
  }
}